sdks/nodejs/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google-cloud/agones-sdk",
-  "version": "1.7.0",
```

Comment:
I don't think we want this part changed. Since I don't think this is editable, I'm going to close this and manually open a PR.